Resources/public/templates/translateaction.hbt

```diff
@@ -18,16 +18,16 @@
             <ul class="ez-contenttranslations-list">
                 {{#each translations}}
                 <li class="ez-contenttranslation">
-                    <a class="ez-contenttranslation-view-link" href="{{ path "viewLocation" id=../location.id languageCode=this }}">
-                        {{ language_name this }}
+                    <a class="ez-contenttranslation-view-link" href="{{ path "viewLocation" id=../location.id languageCode=language }}">
+                        {{ language_name language }}
                     </a>
 
                     <span class="ez-contenttranslation-actions">
-                        <a class="ez-contenttranslation-edit-link" href="{{ path "editContent" id=../content.id languageCode=this }}">
+                        <a class="ez-contenttranslation-edit-link" href="{{ path "editContent" id=../content.id languageCode=language }}">
                             {{translate 'translateaction.edit' 'bar'}}
                         </a>
-                        {{#if ../canDeleteTranslations}}
```

Comment:
I think, change this into something like: `../canDeleteTranslations && ../content.mainLanguageCode !== this` should be enough for disabling the deleting main language code.

Reply:
That was also my first idea but unfortunately this isn't possible to achieve this way. In order to do so, I'll need to create some helpers to implement logical operators `&&` or `!==`.

Reply:
The helper is fine for me. The other solution would be to do `if` inside `if`, but please don't ;)